File: Source/Addins/Eto.Addin.XamarinStudio/Templates/Projects/FS/MainForm.jxeto.fs
namespace ${Namespace}

open System
open Eto.Forms
open Eto.Drawing
open Eto.Serialization.$if$($UseXeto$==True)Xaml$endif$$if$($UseJeto$==True)Json$endif$

type MainForm() as this = 
    inherit Form()
    do 
        $if$($UseXeto$==True)Xaml$endif$$if$($UseJeto$==True)Json$endif$Reader.Load(this)

    member this.HandleClickMe(sender:Object, e:EventArgs) =
        ignore(MessageBox.Show("Hello!"))

    member this.HandleQuit(sender:Object, e:EventArgs) =
        Application.Instance.Quit()

File: src/Addins/Eto.Addin.XamarinStudio/Templates/Projects/CS/MainForm.jxeto.cs
using System;
using System.Collections.Generic;
using Eto.Forms;
using Eto.Drawing;
using Eto.Serialization.$if$($UseXeto$==True)Xaml$endif$$if$($UseJeto$==True)Json$endif$;

namespace ${Namespace}
{	
	public class ${EscapedIdentifier} : Form
	{	
		public ${EscapedIdentifier}()
		{
			$if$($UseXeto$==True)Xaml$endif$$if$($UseJeto$==True)Json$endif$Reader.Load(this);
		}

		protected void HandleClickMe(object sender, EventArgs e)
		{
			MessageBox.Show("I was clicked!");
		}

		protected void HandleQuit(object sender, EventArgs e)
		{
			Application.Instance.Quit();
		}
	}
}
